.github/workflows/bencher.yml

```diff
@@ -140,6 +199,8 @@ jobs:
         continue-on-error: true
         run: |
           ./etc/ci/bencher.py merge ${{ env.SERVO_FILE_SIZE_RESULT }} ${{ env.SERVO_STRIPPED_FILE_SIZE_RESULT }} ${{ env.SERVO_SPEEDOMETER_RESULT }} ${{ env.SERVO_DROMAEO_RESULT }} --bmf-output b.json
+          selected_runner_host='${{ needs.runner-select.outputs.selected-runner-host }}'
+          testbed=${selected_runner_host:-'${{ needs.runner-select.outputs.selected-runner-label }}'}
```

Comment:
Do the benchmark runners differ greatly from another? Imho we should have a graph on main, that allows us to compare results on the same machine (same kind, not necessirily same instance), so that we don't have gaps, where we can't compare. 
Or maybe to ask differently, could we get such a graph on bencher, if we have `ciX` as the testbed?

Reply:
good point! runners on ci3 and ci4 should be identical. while we could ask bencher for a chart with both ci3 and ci4, the chart would have two lines, which doesn’t really make sense. let me rework this :)

Reply:
done in cc7dc1bcd57dcadac35bc23533fb2e63b4f940d6